CONTRIBUTING.md

```diff
@@ -34,6 +36,31 @@ cd /path/to/other-npm-package
 yarn link @hubspot/cli
 ```
 
+
+## Testing
+Eensure you are on Node 10.24.1 before running any tests, since that is the version of node that the build step uses. Using [nvm](https://github.com/nvm-sh/nvm) to switch between versions will help speed up development.
```

Comment:
Can we put a link to the `package.json` line referencing `engine` instead of hard-coded version that may get out of date?